rpc/types_test.go

```diff
@@ -52,6 +52,11 @@ func TestBlockNumberJSONUnmarshal(t *testing.T) {
 		16: {`someString`, true, BlockNumber(0)},
 		17: {`""`, true, BlockNumber(0)},
 		18: {``, true, BlockNumber(0)},
+		19: {`88439993`, false, BlockNumber(88439993)},
+		20: {`-1`, true, BlockNumber(0)},
+		21: {`MaxInt64`, true, BlockNumber(0)},
+		22: {`MinInt64`, true, BlockNumber(0)},
+		23: {`MaxUint64`, true, BlockNumber(0)},
```

Comment:
I mean test `18446744073709551615`, not string `MaxUint64`. Also:

- 9223372036854775807, not string `MaxInt64`
- -9223372036854775808, not string `MinInt64`